docs/how-to-guides/new-user-guides/backup-restore-and-disaster-recovery/back-up-restore-usage-guide.md

```diff
@@ -0,0 +1,120 @@
+    - We provided a label [`resources.cattle.io/backup:](http://resources.cattle.io/backup:) true` that when added to a secret in any namespace, will result in it being backed up.
+- Backups are non-mutating
+- Backups are only of the local cluster
+
+### Notes on Restores
+
+For a restore:
+
+- Restore references restoring a backup to the same cluster it was taken from. This can be with rancher manager installed (*prune must be enabled*) or with it not installed (*no special instructions*).
+- One thing to note when restoring is that you may find yourself needing to “wipe” the cluster of any rancher resources. This can be done by deploying the rancher cleanup script as a job to the cluster. This allows you to install rancher backups again and restore to a completely fresh cluster.
+    - https://github.com/rancher/rancher-cleanup
+    - Make sure to use kubectl to deploy the scripts
+
+### Notes on Migrations
+
+For a migration we have some more nuance, since we are restoring to a different cluster there are a few things to remember that are commonly missed or forgotten.
+
+- Rancher domain must be the same when migrating. This means your previous cluster’s domain name must now point to the new cluster.
+- Rancher should NOT be running already on the cluster you are migrating to. This can cause many issues with rancher backups and certain rancher services as well.
+- Install the SAME version of Rancher from the backup AFTER the backup has been restored.
+- If you choose to provision the new cluster on a different kubernetes version know that this can cause a wide variety of unsupported behaviors because the k8s api available may be different from the one you took a backup from. This can lead to deprecated resources being restored which will cause issues.
+- Migration is a horizontal translation, not a chance for upgrades.
+
+## Edge Cases and Improper Usage
+
+Below are some examples of some INCORRECT uses or expectations of rancher backups.
+
+- Upgrades
+    - Using rancher backups for upgrading Rancher versions is not a valid use case. The best procedure is to take a backup in the current version. Then upgrade your rancher instance using the instructions [here](https://ranchermanager.docs.rancher.com/getting-started/installation-and-upgrade/install-upgrade-on-a-kubernetes-cluster/upgrades), THEN taking another backup after the upgrade is complete. This way if the upgrade fails you have a backup to restore to, and the second backup will be valid to restore to the upgraded rancher version.
+    - Using rancher backups for upgrading kubernetes versions is not a valid use case either. Because the k8s api and available resources are tied to the version, upgrading using backup restore can lead to issues with misaligned sets of resources which may be deprecated, unsupported, or updated. How to upgrade your cluster version will depend on how it was provisioned however the same format as above is recommended (backup, upgrade, backup).
```

Comment:
```suggestion
    - Using Rancher backups for upgrading Kubernetes versions is not a valid use case either. Because the Kubernetes API and available resources are tied to the version, upgrading using backup restore can lead to issues with misaligned sets of resources which may be deprecated, unsupported, or updated. How to upgrade your cluster version will depend on how it was provisioned however the same format as above is recommended (backup, upgrade, backup).
```